api/docs/release.dox

```diff
@@ -233,6 +233,8 @@ Further non-compatibility-affecting changes include:
    DR's decoder.
  - Added decode_sizeof_ex() and instr_get_rel_data_or_instr_target() handling
    relative instruction references.
+ - Added drreg_restore_all() to restore all unreserved registers currently
```

Comment:
See my comment in the issue: drreg_restore_app_values() already exists.

Reply:
That is granular to an operand though. The difference is that this PR restores all unreserved registers currently maintained by drreg.

Reply:
Got it.